tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java

```diff
@@ -907,4 +919,134 @@ public static void saveProperties( Properties props, File bladeSettings )
         }
     }
 
+    public static List<IFile> getDefaultLanguagePropertiesFromModuleProject( IProject project )
```

Comment:
Yeah, but having a property key and value is just a thinly disguised
version of a method call without all the static binding help.  Lets go with
the interface instead.

On Thu, Mar 24, 2016 at 12:57 AM, Andy Wu <notifications@github.com> wrote:

> In
> tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java
> <https://github.com/gamerson/liferay-ide/pull/1457#discussion_r57277386>:
>
> > @@ -907,4 +919,134 @@ public static void saveProperties( Properties props, File bladeSettings )
> >          }
> >      }
> >
> > +    public static List<IFile> getDefaultLanguagePropertiesFromModuleProject( IProject project )
>
> Hey , Greg , I analysed the code about ILiferayProject and its
> implementations and I found the
> String getProperty( String key, String defaultValue );
> method is just for getting theme related property , so we can reuse it for
> getting properties in components (configured via java annotation ) and it
> will be simple without adding new interface.
> what is your opinion ?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1457/files/773d3144328fc2a95d7c4e22e99fc7d02252e646#r57277386>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com